src/viur/core/skeleton.py

```diff
@@ -1220,6 +1228,8 @@ def write(
                 return cls.toDB(skel, update_relations=update_relations)
 
         assert skel.renderPreparation is None, "Cannot modify values while rendering"
+        if skel.is_readonly:
+            raise ValueError("Cannot modify wirte data when the skeleton is readonly")
```

Comment:
A readonly-flagged skeleton can still be modified and written by program code.
The idea of a readonly-skeleton is just that it cannot be read from client, as all bones are readonly.
```suggestion
```